README.rst

```diff
@@ -73,23 +73,33 @@ Clone the Repository
 
 To clone this repository you may use the following: ::
 
-    git clone git@github.com:xmos/lib_xcore_math 
-
+    git clone git@github.com:xmos/lib_xcore_math
 
 Including lib_xcore_math in External Applications
 -------------------------------------------------
 
+``lib_xcore_math`` can be compiled for both x86 platforms and xs3a processors.
+
+On x86 platforms you can develop DSP algorithms and test them for functional correctness;
+this is an optional step before porting the library to an xs3a device.
+
+.. note::
+
+   The instructions in this section refer to the compilation for xs3a; the steps to compile for the x86 platforms are the same,
+   except that the XTC build tools are not necessary and the default toolchain must be used. This can be done by skipping the option
+   ``-DCMAKE_TOOLCHAIN_FILE=../etc/xmos_cmake_toolchain/xs3a.cmake``  when comfiguring CMake.
```

Comment:
this can be `--toolchain=`, `-DCMAKE_TOOLCHAIN_FILE` looks uglier

Reply:
I updated it.